src/__deprecated__/components/checkbox/__spec__.js

```diff
@@ -12,6 +12,7 @@ describe('Checkbox', () => {
       <Checkbox
         name='checkbox'
         label='checkbox'
+        checked={ true }
```

Comment:
Does this need to be assigned a value? When it's set to `true` isn't it preferable to just use `checked` without an assignment?

Reply:
Good point. I'll fix this.

This should've been picked up by the [`react/jsx-boolean-value`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md) ESLint rule. This is yet another occurrence of the FE-1870 issue :disappointed:

Reply:
Fixed in 42fda8d :thumbsup: